versionInfo.gradle

```diff
@@ -1,5 +1,20 @@
 group = 'com.workday.warp'
 
+
+println project.findProperty("scalaVersion")
+
+
+
+def gatlingVersion() {
+    String scalaSuffix = project.findProperty("scalaSuffix")
+
+    if (scalaSuffix == '_2.11') return '2.2.5'
```

Comment:
Couldn't this be simplified to:
```
if (scalaSuffix == '_2.12') return '2.3.1'
else return '2.2.5'
```
???

Reply:
yeah, im not too happy with this. i wanted to make it clear that theres a different gatling version for each scala version, maybe i should just remove the else entirely, or throw an error in that case, or just return something degenerate like empty string. i feel like there at least needs to be a warning printed here.

i dont think it would work to use 2.2.5 with scala 2.10 anyway, and definitely not with any 2.13 milestone releases

Reply:
hmm...I think throwing an UnsupportedVersionException would be cleanest. I could see somebody having an out of date scala version that could cause a major headache trying to get started initially. If we throw an exception then it would be pretty clear it's a Scala version issue.

Reply:
thats a good idea. its complicated a bit because this versionInfo file is used both in buildscript dependencies and build dependencies, and there won't be a defined `scalaVersion` when buildscript dependencies are being evaluated, i think to get the exception to work, ill have to set that property ad hoc somewhere only for buildscript dependencies